routes/web.php

```diff
@@ -17,6 +18,17 @@
     return view('welcome');
 });
 
+Route::get('/pharmacies', [PharmacyController::class, 'index']);
+
+Route::middleware(['auth:sanctum', 'verified'])->group(function () {
+    Route::get('/pharmacies/create', [PharmacyController::class, 'create']);
+    Route::post('/pharmacies', [PharmacyController::class, 'store']);
+    Route::get('/pharmacies/{pharmacy}', [PharmacyController::class, 'show']);
+    Route::patch('/pharmacies/{pharmacy}', [PharmacyController::class, 'update']);
+    Route::delete('/pharmacies/{pharmacy}', [PharmacyController::class, 'destroy']);
+    Route::get('/pharmacies/{pharmacy}/edit', [PharmacyController::class, 'edit']);
```

Comment:
Since you TDD'd the entire resource, you can replace this
```suggestion
    Route::resource('pharmacies', PharmacyController::class);
```